efficientdet/keras/eval.py

```diff
@@ -15,36 +15,53 @@
 # ==============================================================================
 """Eval libraries."""
 from absl import app
+from absl import flags
 from absl import logging
 import tensorflow as tf
 
 import coco_metric
 import dataloader
 import hparams_config
+import utils
 
 from keras import anchors
 from keras import efficientdet_keras
 from keras import postprocess
 
+flags.DEFINE_string('validation_file_pattern', None,
+                    'Glob for evaluation tfrecords (e.g., COCO val2017 set)')
+flags.DEFINE_string(
+    'val_json_file', None,
+    'COCO validation JSON containing golden bounding boxes.')
+flags.DEFINE_string('checkpoint', None, 'Location of the checkpoint to evaluate.')
+flags.DEFINE_string('model_name', 'efficientdet-d0',
+                    'Model name: the efficientdet model to use.')
+
+flags.mark_flag_as_required('validation_file_pattern')
```

Comment:
You need to move this right after "if __name__ == '__main__'"

Reply:
Done.